godot_project/editor/controls/dockers/workspace_docker/simulations_docker/simulations_docker_controls/atomic_structure_model_validator.gd

```diff
@@ -18,6 +18,7 @@ var _thread: Thread
 var _overlaps: Dictionary[OverlapData, int] = {} # Overlap data : Alert ID
 var _data: Dictionary[Metadata, int] = {} # Data : Alert ID
 var _workspace_context: WorkspaceContext = null
+var _last_validation_frame: int = -1
```

Comment:
This kinda looks like a `_first_run=true` or `_has_ever_validated=false` boolean